web-ui/src/main/resources/catalog/templates/top-toolbar.html

```diff
@@ -79,6 +79,11 @@
               <i class="fa fa-fw fa-pencil"></i>&nbsp;<span translate>batchEditing</span>
             </a>
           </li>
+          <li ng-if="user.isAdministratorOrMore() && healthCheck.IndexHealthCheck === true">
```

Comment:
If the tool is restricted to administrators, why is it located in the Contribute section? Maybe it would make sense to have it in the admin console.

Reply:
Ifremer may wants to open it to group reviewers but for now, we only decided to have it for admin. So it is for that reason.